fusionauth-netcore-client/src/io/fusionauth/DefaultRESTClient.cs

```diff
@@ -57,6 +58,10 @@ class DefaultRESTClient : IRESTClient {
       httpClient = new HttpClient {BaseAddress = new Uri(host)};
```

Comment:
```suggestion
      var handler = new HttpClientHandler
      {
        UseCookies = false,
      };
      httpClient = new HttpClient(handler) {BaseAddress = new Uri(host)};
```

In the same vein as @matt-lethargic's comment regarding preventing the accidental reuse of cookies. While there's not an easy way to override the `HttpClientHandler` for the custom HTTP client provided by the end-user, we can at least ensure that the default client will not re-use cookies by disabling the functionality during initial construction.